pkg/crds/operator/operator.tigera.io_installations.yaml

```diff
@@ -12708,6 +12708,9 @@ spec:
                 - Calico
                 - TigeraSecureEnterprise
                 type: string
+              version:
```

Comment:
May not be necessary, see my comment on getCalicoVersion.

Reply:
As discussed we need this field to sequence component upgrades.